src/components/FilterBar.test.jsx

```diff
@@ -0,0 +1,47 @@
+import { render, screen } from '@testing-library/react';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import FilterBar from './FilterBar';
+
+const renderWithTheme = (ui) =>
+  render(<ThemeProvider theme={createTheme()}>{ui}</ThemeProvider>);
+
+const setQueryParam = () => {};
+
+describe('FilterBar', () => {
+  test('SortBy filter (deadline.asc) rendered', () => {
```

Comment:
nit: prefer to use the active tense in test names. E.g. `renders SortBy filter (deadline.asc)`